podpac/core/data/test/test_datasource.py

```diff
@@ -330,26 +327,13 @@ def test_evaluate_missing_dims(self):
             node.eval(Coordinates([1], dims=["time"]))
 
     def test_evaluate_crs_transform(self):
```

Comment:
@mpu-creare I took most of this out so that it was just testing the datasource part. I'll make sure to add this test back into the interpolation tests somewhere if it covers a case that isn't cover there already.

Reply:
Great! Thanks for actually looking carefully at the whole test suite.